decisions/Rhovanion.txt

```diff
@@ -566,6 +567,90 @@ political_decisions = {
 	}
 }
 
+# Culture acceptance
+political_decisions = {
+	accept_avari_RHO = {
+		picture = "kingdom_of_dorwinion"
+		potential = {
+			tag = RHO
+			population_2 = 1
+			education_1 = 1
+			NOT = { accepted_culture = avari }
+			OR = {
+				DOR_539 = { owned_by = THIS }
+				DOR_545 = { owned_by = THIS }
+				DOR_550 = { owned_by = THIS }
+				DOR_555 = { owned_by = THIS }
+				DOR_558 = { owned_by = THIS }
+			}
+		}
+		
+		allow = {
+			DOR_539 = { owned_by = THIS }
+			DOR_545 = { owned_by = THIS }
+			DOR_550 = { owned_by = THIS }
+			DOR_555 = { owned_by = THIS }
+			DOR_558 = { owned_by = THIS }
+			population_4 = 1
+			education_2 = 1
+			magic_1 = 1
+			culture_2 = 1
+			money = 10000
```

Comment:
At this point in the game, money really shouldn't be too much of an object -- either have the money be like 100k, or no money